app/src/main/java/com/example/keniphhomemaintenance/dwellings/DwellingsViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.keniphhomemaintenance.dwellings
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+
+class DwellingsViewModel : ViewModel() {
+
+    private val placeholderDwelling1: Dwelling =
```

Comment:
`placeholderDwelling1` and `placeholderDwelling2` could be removed eventually once data is stored (though thinking down the line it might be okay to have 1 or 2 "sample" dwellings

Reply:
Stop resolving context-providing comments